manim/utils/cfg_subcmds.py

```diff
@@ -0,0 +1,187 @@
+                                console.print(
+                                    "[red bold]Invalid value. Try again.[/red bold]")
+                                console.print(
+                                    f"Enter the style for {key}:", style=key, end="")
+                                temp = input()
+                            else:
+                                default[key] = temp
+            config[category] = dict(default)
+    else:
+        action = "open"
+
+    if level is None:
+        console.print(
+            f"Do you want to {action} the default config for this User?(y/n)[[n]]",
+            style="dim purple",
+            end="",
+        )
+        action_to_userpath = input()
+    else:
+        action_to_userpath = ""
+
+    if action_to_userpath.lower() == "y" or level=="user":
+        cfg_file_path = os.path.join(
+            guarantee_existence(
+                os.path.dirname(config_paths[1])
+                ),"manim.cfg")
+        console.print(USER_CONFIG_MSG)
+    else:
+        cfg_file_path = os.path.join(
+            guarantee_existence(
+                os.path.dirname(config_paths[2])
+                ),"manim.cfg")
+        console.print(CWD_CONFIG_MSG)
+    with open(cfg_file_path, "w") as fp:
+        config.write(fp)
+    if openfile:
+        open_file(cfg_file_path)
+
+def show():
+    current_config = finalized_configs_dict()
+    for category in current_config:
+        console.print(f"{category}",style="bold green underline")
+        for entry in current_config[category]:
+            if category=="logger":
+                console.print(f"{entry} :",end="")
+                console.print(
+                    f" {current_config[category][entry]}",
+                    style=current_config[category][entry]
+                    )
+            else:
+                console.print(f"{entry} : {current_config[category][entry]}")
+        console.print("\n")
+
+def export(path):
+    config = _run_config()[1]
+    if os.path.abspath(path) == os.path.abspath(os.getcwd()):
+        console.print(
+            """You are reading the config from the same directory you are exporting to.
+This means that the exported config will overwrite the config for this directory.
+Are you sure you want to continue?[y/n]""",
+            style="red bold", end=""
+            )
+        proceed = True if input().lower()=="y" else False
+    else:
+        proceed = True
+    if proceed:
+        if not os.path.isdir(path):
+            console.print(f"Creating folder: {path}.",style="red bold")
+            os.mkdir(path)
+        with open(os.path.join(path,"manim.cfg"),"w") as outpath:
+            config.write(outpath)
+            from_path = os.path.join(os.getcwd(),'manim.cfg')
+            to_path = os.path.join(path,'manim.cfg')
+        console.print(f"Exported final Config at {from_path} to {to_path}.")
+    else:
+        console.print("Could NOT write config.", style="red bold")
```

Comment:
**suggestion (if-minor)** : Maybe be more explicit here. This could be misleading, as one could understand that it was a bug instead of the user that typed `n`.

Reply:
Agreed. I've made it say `Aborted...` now.